pkgs/development/python-modules/flit-core/default.nix

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
I think we can remove toml here https://github.com/pypa/flit/blob/main/doc/history.rst#version-36